kernel/packages/shared/catalogs/types.ts

```diff
@@ -16,6 +16,8 @@ export type Wearable = {
   replaces?: string[]
   rarity: RarityEnum
   representations: BodyShapeRepresentation[]
+  i18n: { code: string; text: string }[]
```

Comment:
This were already present in all wearables, but they weren't explicit on the type definition